.github/workflows/release.yaml

```diff
@@ -4,8 +4,6 @@ on:
   push:
     tags:
       - 'v*'
-    branches:
-      - master
```

Comment:
Are you sure about this? 
Won't this start running the release workflow for all branches? 
I think we want it only in the master branch, unless you are doing this for testing purposes.

Reply:
I think you're right, nice catch. I'll set it so that the workflow only triggers on pushes of branches of the form `v*`.

Reply:
This is a confusing part of github actions. I double checked, and the `tags` and `branches` fields work in an OR manner, not AND. Consider the following:
```yaml
  push:
    tags:
      - 'v*'
    branches:
      - master
```
This causes the workflow to be triggered when a tag of the form `v*` is pushed, OR when a commit is pushed to `master`. This isn't what we're looking for.

I agree that it would be better to only trigger this workflow if a tag of the form `v*` was pushed to the `master` branch specifically, but I don't see a way to do this. In practice, it is not likely that we will accidentally push a tag of this form that is not pointing at a commit on `master`.

Reply:
Ok understood.